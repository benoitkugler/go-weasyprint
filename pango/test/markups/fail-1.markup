<span foo="bar">blah</span>
