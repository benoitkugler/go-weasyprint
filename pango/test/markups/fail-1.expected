ERROR: attribute 'foo' is not allowed on the <span> tag